.github/workflows/ci.yml

```diff
@@ -11,14 +11,18 @@ on:
 jobs:
   tests:
     runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node: [ 20, 24 ]
```

Comment:
[nit]: I understand that these changes are temporary and will be removed, but let's remove the extra spaces for the clear commit history.

Reply:
> [nit]: I understand that these changes are temporary and will be removed, but let's remove the extra spaces for the clear commit history.

removed it, thx!